tsconfig.json

```diff
@@ -18,6 +18,7 @@
     //     "*": ["typings/*"]
     // },
     "esModuleInterop": true,
+    "resolveJsonModule": true,
```

Comment:
This could be broken for ESM later (although I know it is transformed as commonjs), but it could be better to not adding such unstandard config.